ironfish-cli/src/commands/fees.ts

```diff
@@ -29,17 +29,17 @@ export class FeeCommand extends IronfishCommand {
     const feeRates = await client.estimateFeeRates()
 
     this.log('Fee Rates ($ORE/kB)')
-    this.log(`low:    ${feeRates.content.low || ''}`)
-    this.log(`medium: ${feeRates.content.medium || ''}`)
-    this.log(`high:   ${feeRates.content.high || ''}`)
+    this.log(`slow:    ${feeRates.content.slow || ''}`)
+    this.log(`average: ${feeRates.content.average || ''}`)
+    this.log(`fast:   ${feeRates.content.fast || ''}`)
```

Comment:
```suggestion
    this.log(`fast:    ${feeRates.content.fast || ''}`)
```

nit - to align the rate values